client/lib/post-normalizer/index.js

```diff
@@ -210,6 +210,7 @@ function isCandidateForContentImage( imageUrl ) {
 
 const bannedUrlParts = [
 	'feeds.feedburner.com',
+	'feeds.wordpress.com/',
```

Comment:
nope, intended. really need to redo this whole block to be a bit more discriminate.